src/app/containers/ComscoreAnalytics/Canonical/index.jsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable jsx-a11y/alt-text */
+import React, { useContext, useEffect } from 'react';
+import { Helmet } from 'react-helmet';
+import { UserContext } from '#contexts/UserContext';
+
+const CanonicalComscoreAnalytics = () => {
+  const { personalisationEnabled } = useContext(UserContext);
+
+  const staticAssetsPath = `${process.env.SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN}${process.env.SIMORGH_PUBLIC_STATIC_ASSETS_PATH}`;
+
+  useEffect(() => {
+    const csUcfr = personalisationEnabled ? '1' : '0';
+    /* eslint-disable no-underscore-dangle */
+    window._comscore = window._comscore || [];
+    window._comscore.push({ c1: '2', c2: '17986528', cs_ucfr: csUcfr });
+  }, [personalisationEnabled]);
+
+  return (
+    <Helmet>
+      <script
+        async
+        type="text/javascript"
+        src={`${staticAssetsPath}static/js/comscore-1.0.js`}
```

Comment:
This looks good - just one question around versioning. Should we consider how we're going to update the script, and how we can keep track of updated?

Reply:
Ah sorry, just saw @DenisHdz just mentioned this also